aten/src/ATen/native/cpu/ScatterGatherKernel.cpp

```diff
@@ -655,9 +655,9 @@ void cpu_scatter_reduce_expanded_index(const Tensor& self, const Tensor& index,
     for (const auto i : c10::irange(begin, end)) {
       int64_t index = index_data[i];
       TORCH_CHECK(index >= 0 && index < index_upper_bound,
-                "index ", index,
-                " is out of bounds for dimension ", 0,
-                " with size ", index_upper_bound);
+                  "index ", index,
+                  " is out of bounds for dimension ", 0,
+                  " with size ", index_upper_bound);
```

Comment:
Nit: would be nice to keep formatting changes outside of functional ones (also, can we please make sure that file is covered by our standard clang-format linter, this way it will be uniform for all participants

Reply:
oh write, this should go with the previous PR. I must have distorted this when rebasing ...